ci-operator/config/devfile/integration-tests/devfile-integration-tests-master.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-integration
+  steps:
+    cluster_profile: aws
+    test:
+    - as: devworkspaces-operator-e2e
+      commands: |
+        export PATH=$PATH:/tmp/bin
+        mkdir /tmp/bin
+        curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.5/linux/oc.tar.gz | tar xvzf - -C /tmp/bin/ oc
```

Comment:
I recommend using this instead: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

Reply:
> I recommend using this instead: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

Updated as recommended.

Reply:
okay. Other than that you will need to join the openshift github org, otherwise you can't be present in the OWNERS files

Reply:
Do you know who can help me to join the openshift github org?

Reply:
https://source.redhat.com/groups/public/atomicopenshift/atomicopenshift_wiki/openshift_onboarding_checklist_for_github